testing/testpb/test_grpc.pb.go

```diff
@@ -4,7 +4,6 @@ package testpb
 
 import (
 	context "context"
-
```

Comment:
Can you undo these generated file changes please? I don't expect them to change in this PR.

Reply:
Reverted these changes.